src/main/scala/cosas/ops/typeSets/Conversions.scala

```diff
@@ -107,88 +107,3 @@ object ToMap {
     def apply(s: D :~: T): Out =  rest(s.tail) + (key -> s.head.value)
```

Comment:
I know, but I think that it's better to reduce number of different things, because this can be easily done with serialise.